integration-tests/tests/src/hooks/open-realm-before.ts

```diff
@@ -35,9 +36,32 @@ export function openRealmHook(config: OpenRealmConfiguration = {}) {
     if (this.realm) {
       throw new Error("Unexpected realm on context, use only one openRealmBefore per test");
     } else {
-      const result = await openRealm(config, this.user);
-      this.realm = result.realm;
-      this.config = result.config;
+      this.closeRealm = async () => {
```

Comment:
So this would be invoked if `openRealm` threw an error or timed out?

Reply:
Correct.